packages/amplify_core/ios/amplify_core.podspec

```diff
@@ -0,0 +1,23 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint amplify_core.podspec' to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'amplify_core'
+  s.version          = '0.0.1'
+  s.summary          = 'A base package shared across Amplify Flutter library.'
+  s.description      = <<-DESC
+A base package shared across Amplify Flutter library.
+                       DESC
+  s.homepage = 'https://github.com/aws-amplify/amplify-flutter'
+  s.license = 'Apache License, Version 2.0'
+  s.author = { 'Amazon Web Services' => 'amazonwebservices' }
+  s.source = { :git => 'https://github.com/aws-amplify/amplify-flutter.git' }
+  s.source_files = 'Classes/**/*'
+  s.dependency 'Flutter'
+  s.platform = :ios, '9.0'
```

Comment:
Aren't we using a minimum of 11.0?

Reply:
in your app, Cocoapods (?) will resolve to the max of the min dependency. This merely says that this package doesn't need anything more than 9.0 so other packages at 9.0 can use it as well.